src/patternfly/components/ClipboardCopy/docs/code.md

```diff
@@ -0,0 +1,30 @@
+## Overview
+
+## Accessibility
+
+| Attribute | Applied To | Outcome |
+| -- | -- | -- |
+| `aria-label="Show Contents"` | `.pf-c-clipboard-copy__group-toggle` |  Provides an accessible name for the button when an icon is used instead of text. **Required when an icon is used with no supporting text.** |
+| `aria-label="Copy to Clipboard"` | `.pf-c-clipboard-copy__group-copy` |  Provides an accessible name for the button when an icon is used instead of text. **Required when an icon is used with no supporting text.** |
```

Comment:
These two aria-label attributes also appear to be overlooked in your latest commit. 

Also, in the handlebar you have `aria-label="Show content"` (singular) which is also consistent with one of @stacymcauliffe's suggestions.